src/Sylius/Behat/Context/Ui/Admin/ManagingTaxCategoryContext.php

```diff
@@ -91,4 +99,193 @@ public function iShouldBeNotifiedAboutSuccessfulDeletion()
             'Successful deletion message does not appear'
         );
     }
+
+    /**
+     * @Given I want to create a new tax category
+     */
+    public function iWantToCreateNewTaxCategory()
+    {
+        $this->taxCategoryCreatePage->open();
+    }
+
+    /**
+     * @When I specify its code as :code
+     * @When I do not specify its code
+     */
+    public function iSpecifyItsCodeAs($code = '')
+    {
+        $this->taxCategoryCreatePage->specifyCode($code);
+    }
+
+    /**
+     * @When I name it :name
+     * @When I rename it to :name
+     * @When I do not name it
+     * @When I remove its name
+     */
+    public function iNameIt($name = null)
+    {
+        $this->taxCategoryCreatePage->nameIt($name);
+    }
+
+    /**
+     * @When I add it
+     * @When I try to add it
+     */
+    public function iAddIt()
+    {
+        $this->taxCategoryCreatePage->create();
+    }
+
+    /**
+     * @Then the tax category :taxCategory should appear in the registry
+     */
+    public function thisTaxCategoryShouldAppearInTheRegistry(TaxCategoryInterface $taxCategory)
```

Comment:
We should find it on the index I think, that will be the simplest solution. And find it by name only.